xonsh/history/sqlite.py

```diff
@@ -12,6 +12,10 @@
 from xonsh.history.base import History
 import xonsh.tools as xt
 
+CACHE = threading.local()
+TABLE_NAME = "xonsh_history"
+CREATED_SQL_TBL = "CREATED_SQL_TABLE"
```

Comment:
Please use more unique names here, since this sub-packages is using amalgamate.  For example, something like `XH_SQLITE_CACHE`